pkg/logs/input/file/file_provider.go

```diff
@@ -52,6 +47,16 @@ func (t *File) getSourceIdentifier() string {
 	return ""
 }
 
+// GetScanKey returns a key used by the scanner to index the scanned file.
+// If it is a file scanned for a container, it will use the format: <filepath>/<container_id>
+// Otherwise, it will simply use the format: <filepath>
+func (t *File) GetScanKey() string {
```

Comment:
Can `container_id` be set by the user to a friendly name (For example `Test`)? If yes `<filepath>/Test` can match a filename outside a container. I am not sure if it is an issue as it is unlikely.

Reply:
The container ID (= Source.Config.Identifier) is set by the autodiscovery mechanism, so it's unlikely there will be a a existing file matching the output of  `GetScanKey`.